test/test.js

```diff
@@ -20,6 +20,18 @@ function makePOJO(ast) {
   return JSON.parse(JSON.stringify(ast));
 }
 
+function readOptionsFromFile(file) {
+  var s = fs.readFileSync(file, 'utf-8');
+  var lines = s.split(/\n/, 20);
```

Comment:
didn't know that String.prototype.split had a second param. Cool! Learned something today